AdvanceSwiftExercise.playground/Pages/Class.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,188 @@
+//1
+class Emp {
+  var ID = ""
+  var age = 0
+}
+
+var emp = Emp()
+emp.ID = "123"
+emp.age = 20
+print("name:",emp.ID)
+print("age:",emp.age)
+
+//2
+class Student {
+    var name: String?
+    var college: String?
+    var department: String?
+    
+    init(name: String, college: String) {
+        self.name = name
+        self.college = college
+    }
+    
+    init(name: String, department: String) {
+        self.name = name
+        self.department = department
+    }
+}
+
+let s1 = Student(name: "Mansi", college: "vgec")
+let s2 = Student(name: "Mansi", department: "Computer")
+print("Name \(s1.name!), college: \(s1.college!)")
+print("Name \(s2.name!), Department: \(s2.department!)")
```

Comment:
Don't use force unwrapping, it will not affect here but practice to use if let or guard let for future.